lib/ansible/modules/crypto/luks_device.py

```diff
@@ -153,6 +168,26 @@
   luks_device:
     device: "/dev/loop0"
     state: "absent"
+
+- name: create a container with label
+  luks_device:
+    device: "/dev/loop0"
+    state: "present"
+    keyfile: "/vault/keyfile"
+    label: personalLabelName
+
+- name: open the LUKS container based on label or UUID without device; name it "mycrypt"
```

Comment:
```suggestion
- name: open the LUKS container based on label without device; name it "mycrypt"
```
You don't specify `uuid` here.

Reply:
You are correct, thanks!